shared/cfg/guest-hw.cfg

```diff
@@ -460,6 +460,10 @@ variants:
             restore_ovmf_vars = yes
             Windows:
                 send_key_at_install = ret
+	block_with_iommu:
+            restore_ovmf_vars = yes
+            Windows:
+                send_key_at_install = ret
```

Comment:
The case `block_with_iommu` configure should be aligned with `unattended_install` since it belongs to `ovmf` scenario.

Reply:
updated ,thanks


 (1/2) Host_RHEL.m8.u3.product_av.ovmf.qcow2.virtio_blk.up.virtio_net.Guest.Win2016.x86_64.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.default_install.aio_threads.q35: PASS (879.90 s)
 (2/2) Host_RHEL.m8.u3.product_av.ovmf.qcow2.virtio_blk.up.virtio_net.Guest.Win2016.x86_64.io-github-autotest-qemu.block_with_iommu.with_installation.cdrom.extra_cdrom_ks.q35: PASS (750.79 s)